src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Locale;
+
 public class FileWork {
+    private static final String REGEX = "\\s*(\\s|,|\\?|!|\\.)\\s*";
+    private static final String FIRST_LETTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        File file = new File(fileName);
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file))) {
+            String line = bufferedReader.readLine();
+            StringBuilder builder = new StringBuilder();
```

Comment:
you can move builder declaration outside try block, the same with return createList(builder)